jvcl/run/JvAppIniStorage.pas

```diff
@@ -704,8 +706,21 @@ procedure TJvCustomAppIniStorage.DeleteSubTreeInt(const Path: string);
           IniFile.EraseSection(Sections[I])
       else
         for I := 0 to Sections.Count - 1 do
-          if Pos(TopSection, Sections[I]) = 1 then
-            IniFile.EraseSection(Sections[I]);
+        begin
+          // --- start bugfix ---
```

Comment:
That type of comment is really not necessary, it clutters the source code too much.